en/docs/references/about-this-release.md

```diff
@@ -98,6 +98,13 @@ The following new features and enhancements (introduced in WSO2 IS 6.0.0) are in
 
     [Learn more]({{base_path}}/deploy/configure-recaptcha/)
 
+-   **OAuth client ID tenant unification**
+
+    OAuth consumer application client IDs are now tenant unique allowing same client ID to exist in multiple tenants. This feature requires enabling tenant qualified URLs and tenanted sessions which are by default enabled in IS 7.0.0 onwards.
```

Comment:
TODO: Need to add documentation on steps to disable these configs. I.e. have to modify the DB constraints

Reply:
Add a <client_id> unique key constraint to the consumer apps table

Reply:
done